modules/ROOT/pages/govcloud-data-control.adoc

```diff
@@ -0,0 +1,98 @@
+= GovCloud Data Control
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+MuleSoft Government Cloud (GovCloud) organizations can restrict the data entities exposed to their Salesforce Community Cloud ACM instance.
+
+Restrictions are defined on the Anypoint organization management plane and applied at the data bridge level.
```

Comment:
GLOBAL: never use "Anypoint" alone. Search for and fix.

Reply:
Fixed.